e2e/__test__/tours.test.js

```diff
@@ -0,0 +1,186 @@
+});
+
+getForecast.mockResolvedValue({
+  time: new Date(),
+  forecast: 'sad weather'
+});
+
+describe('tours api', () => {
+  beforeEach(() => {
+    return db.dropCollection('tours');
+  });
+
+  const testTour = {
+    title: 'Testing Tour',
+    activities: ['Test1', 'Test2', 'Test3'],
+    launchDate: new Date(),
+    stops: [{}]
+  };
+
+  const testLocation = {
+    name: 'Rontoms',
+    address: '600 E Burnside St, Portland, OR'
+  };
+
+  const testAttendance = {
+    attendance: 24
+  };
+
+  function postTour(tour) {
+    return request
+      .post('/api/tours')
+      .send(tour)
+      .expect(200)
+      .then(({ body }) => body);
+  }
+
+  it('posts a tour', () => {
+    return postTour(testTour).then(tour => {
+      expect(tour).toMatchInlineSnapshot(
+        {
+          _id: expect.any(String),
+          launchDate: expect.any(String),
+          stops: [
+            {
+              _id: expect.any(String)
+            }
+          ]
+        },
+        `
+        Object {
+          "__v": 0,
+          "_id": Any<String>,
+          "activities": Array [
+            "Test1",
+            "Test2",
+            "Test3",
+          ],
+          "launchDate": Any<String>,
+          "stops": Array [
+            Object {
+              "_id": Any<String>,
+            },
+          ],
+          "title": "Testing Tour",
+        }
+      `
+      );
+    });
+  });
+
+  it('gets all tours', () => {
+    return Promise.all([
+      postTour(testTour),
+      postTour(testTour),
+      postTour(testTour)
+    ])
+      .then(() => {
+        return request.get('/api/tours').expect(200);
+      })
+      .then(({ body }) => {
+        expect(body.length).toBe(3);
+        expect(body[0]).toMatchInlineSnapshot(
+          {
+            _id: expect.any(String),
+            launchDate: expect.any(String),
+            stops: [
+              {
+                _id: expect.any(String)
+              }
+            ]
+          },
+          `
+          Object {
+            "__v": 0,
+            "_id": Any<String>,
+            "activities": Array [
+              "Test1",
+              "Test2",
+              "Test3",
+            ],
+            "launchDate": Any<String>,
+            "stops": Array [
+              Object {
+                "_id": Any<String>,
+              },
+            ],
+            "title": "Testing Tour",
+          }
+        `
+        );
+      });
+  });
+
+  it('adds a stop to the tour', () => {
+    return postTour(testTour)
+      .then(tour => {
+        return request
+          .post(`/api/tours/${tour._id}/stops`)
+          .send(testLocation)
+          .expect(200)
+          .then(body => {
+            return [body, testLocation, testLocation];
+          });
+      })
+      .then(out => {
+        const stop = out[0].body[1];
+        expect(stop).toMatchInlineSnapshot(
+          {
+            _id: expect.any(String)
+          },
+          `
+          Object {
```

Comment:
why is the stop object only an id? can you run a test that shows the stop that was posted? it's hard to tell whether this route is working.